deploy/k8s/sensor.sh

```diff
@@ -20,7 +20,7 @@ fi
 echo "CLUSTER set to $CLUSTER"
 
 if [[ -z "${ROX_ADMIN_PASSWORD}" ]]; then
-    export ROX_ADMIN_PASSWORD="${ROX_PASSWORD:-}"
+    export ROX_ADMIN_PASSWORD="${ROX_ADMIN_PASSWORD:-}"
```

Comment:
This block seems be a no-op, except in the case `ROX_ADMIN_PASSWORD` is either unset, or set to empty value but not exported. Either way, it does not seem to provide much value?